templates/greenlight/env.j2

```diff
@@ -231,4 +231,4 @@ DB_PASSWORD={{ bbb_greenlight_db_password }}
 #   open - For open registration
 #   invite - For invite only registration
 #   approval - For approve/decline registration
-DEFAULT_REGISTRATION=open
+DEFAULT_REGISTRATION={{ bbb_greenlight_reg | default('open') }}
```

Comment:
Maybe the default value should be set central in `defaults/main.yml`

Reply:
Indeed, although this has already been added in #11 (with the default in the jinja2-template. I would still prefer this to be in the `defaults/main.yml`

Reply:
Ah, now i see: It is already in the defaults.yml so there is no further need to specify a default here...